Test/DatabaseTests/TestDatabaseLoaders.cs

```diff
@@ -505,5 +507,28 @@ public static void IsoformReadTest()
 
             Assert.AreNotEqual(proteinList[2].Accession, proteinList[4].Accession);
         }
+
+        [Test]
+        public static void TestListOfAvailableUniProtProteomes()
+        {
+            Dictionary<string,string> proteomesDictionary = ProteinDbRetriever.UniprotProteomesList();
+            Assert.AreEqual(292464, proteomesDictionary.Keys.Count);
```

Comment:
Do we need to test the exact number? Could we just test that it's greater than 200k entries?

Reply:
no. and you're right not to include a specific number. not sure this really counts as a unit test. actually could use ideas for real unit tests. these are pretty weak.